samples/logger/pom.xml

```diff
@@ -32,6 +32,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Could you please share more context about why the janino dependency is needed?

Reply:
To trigger (enable) conditional processing and prove that the log config with conditions (conditions in limited form of `isDefined` and `isNull`) works ok with native image. Note the log config contains such a condition.